src/Sylius/Component/Product/Model/Product.php

```diff
@@ -351,6 +351,16 @@ public function getVariants()
     /**
      * {@inheritdoc}
      */
+    public function getAllVariants()
```

Comment:
I would use `Criteria` here to avoid lazy loading
http://doctrine-orm.readthedocs.org/projects/doctrine-orm/en/latest/reference/working-with-associations.html#filtering-collections